examples/acquiring_data/downloading_hmi.py

```diff
@@ -32,15 +35,26 @@
 # don't provide a path it will download the file into the sunpy data directory.
 # The output provides the path of the downloaded files.
 # The result can be from several data clients, so we have to index the client first
-# client and then index the file.
+# and then index the file.
 
 # Slice the first record returned by the first client.
-downloaded_file = Fido.fetch(result[0, 0])
+downloaded_file = Fido.fetch(result[0])
 print(downloaded_file)
 
 ###############################################################################
-# Now load it into a map and plot it
+# Now load it into a map and plot it.
+# We see that solar North is pointed down instead of up in this image, which is
+# indicated by the coordinates (that range from positive to negative, rather
+# than negative to positive).
 hmi_map = sunpy.map.Map(downloaded_file[0])
 fig = plt.figure()
 hmi_map.plot()
 plt.show()
+
+###############################################################################
+# Rotate the image such that solar North is pointed up.
+# The order specifies the type of interpolation; in this case, 3 refers to
+# bi-cubic.
```

Comment:
See my above comment. I still think this could use 1 sentence (or phrase) explaining why just calling `rotate` with no specification for the angle removes the roll angle. People may find this mysterious.